### PR TITLE
ADF-151. Fix compatibility with advanced search

### PR DESCRIPTION
**Pull request by voknil**
https://oat-sa.atlassian.net/browse/ADF-151

### CODE REVIEW

File: model/search/GenerisSearch.php [hunk 1]
```diff
@@ -51,7 +51,10 @@ public function query($queryString, $type, $start = 0, $count = 10, $order = 'id
         ]);
         $ids = array();
         foreach ($results as $resource) {
-            $ids[] = $resource->getUri();
+            $ids[] = [
```
Comment (mike-ionut-mihai-sandu-tao) [inline model/search/GenerisSearch.php:54]: do we need to update tests ?

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
please update tests if any available

File: model/search/GenerisSearch.php (hunk 1 above)
Comment (voknil) [reply model/search/GenerisSearch.php:54]: haven't found any affected ones

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: model/search/GenerisSearch.php (hunk 1 above)
Comment (gabrielfs7) [reply model/search/GenerisSearch.php:54]: We should have merged to integration branch till Advanced search is stable @voknil

**Pull request merged**